platform/android/.gitignore

```diff
@@ -11,9 +11,6 @@ build/
 *.so
 *.apk
 
-# Lib assets
-MapboxGLAndroidSDK/src/main/assets/
```

Comment:
not related to this PR but noticed this is an old unused configuration while testing with assetfilesource